### ISSUE TITLE
Can someone please help a poor home-automation noob here?

### ISSUE THREAD

**Issue by ablohme** [wailuen/homebridge-sensibo-sky#19]
Sorry for being a total noob here.

Ok first! I am waiting for my Sensibo Sky Smart Air Conditioner Controller. (Arrives much later today)

Meanwhile I have a Mac mini with homebridge up & running and I have retrieved a API key that I have set in the homebridge config parameter. Now to my million dollar (Ok, $10 dollar to the one who helps me out, I promise!) Question ❓ 

Where do I go from here? Shall I connect the Sensibo to my Mac via Bluetooth and then it will communicate with the Sky Smart Controller via Homebridge automagically?

Or do I need one of these Z-Wave USB dongles? Or is there something else I need to configure, etc?

A little home automation history of me:
I do have a homekit setup today (Without any of the homebridge-fuss) but they are ALL HomeKit compliant and works well, Philips Hue, Elgato Eve temperature thingy, and a nice Glue Smart Lock that works like a dream. 

But when I decided to go for remote controlling my AC using HomeKit, thats when I was thrown into this 🔥 hell-raging-fire 🔥  of advancedness with z-wave routers, bluetooth connection this'n that over my Google search for "HomeKit Panasonic" 

Thank you all very much on beforehand!
Best regards, Alexander Blohmé <alexbleks>

**Comment by Cloudore** [comment]
Homebridge will contact the Sensibo server directly through the API, as long as Homebridge is working correctly, all you need to do is input the API in the config.json. Set up your AC in the Sensibo app, connect the Sensibo to your local wifi, restart homebridge, and it will appear in your default room in the home.app.

I use homebridge and it works the same, if not better, than some of my HomeKit devices, its rock solid.

**Comment by ablohme** [comment]
So I don't need bluetooth or anything on the Homebridge raspberry?

**Comment by wailuen** [comment]
No. It work directly once it is setup properly with the config. Of course, the pod first need to be operational via the sensibo app.

**Comment by ablohme** [comment]
Thanks, but would it technically be possible to communicate via Z-Wave instead of the Sensibo server? I mean, lets say Sensibo goes bankrupt and shuts down their servers, or are having server issues?

**Comment by Cloudore** [comment]
Not possible, you might find an alternative for serverless with broadlink and its homebridge plug in

**Comment by ablohme** [comment]
Cloudore: Thanks, but with a broadlink you won't get any status-feedback? Or does the Panasonic Acs give feedback at all to the remote/sensibo?

**Comment by Cloudore** [comment]
Nor do you get deedback with the Sensibo, if you turn your ac on with your remote, the sensibo never knows.

With both the broadlink and sensibo the state of your ac is saved in a variable, if you change the status with anything other than the the service itself, they become desynced until you manipulate it again through whichever app you choose.

**Comment by wailuen** [comment]
Hi Cloudore, the part that i love about sensibo is that it actually detected the signal from the remote and thus attempt to keep the state in sync. Mine work perfectly this way.
